newm-server/build.gradle.kts

```diff
@@ -57,6 +57,7 @@ dependencies {
     implementation(Dependencies.Ktor.SERIALIZATION_JSON)
     implementation(Dependencies.Ktor.SERVER_STATUS_PAGES)
     implementation(Dependencies.Ktor.SERVER_CORS)
+    implementation(Dependencies.Ktor.SERVER_FORWARDED_HEADER)
```

Comment:
per https://github.com/ktorio/ktor/issues/1112 - ktor verifies that you are using `https` protcol when you set the Secure option on a cookie.  Because CIO does not support `https` we need to add support for forwarding `X-Forwarded-SSL` headers in order to use `Secure` cookies